src/Test/Utilities/Portable/TestResource.resx

```diff
@@ -386,7 +386,7 @@ namespace My
             {
                 *intref = 1;
             }
-            fixed (int* p = stackalloc int[100])
+            fixed (int* p = stackalloc int[] { 100 }, q = stackalloc[] { 100 })
```

Comment:
Good catch. 
@vsadov, this reminds me this file should probably have been updated for the C# 7.2 ref features too.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/24713](https://github.com/dotnet/roslyn/issues/24713) to strengthen AllInOneCSharpCode test

---
In reply to: [166782600](https://github.com/dotnet/roslyn/pull/24249#discussion_r166782600) [](ancestors = 166782600)